.eslintcache

```diff
@@ -0,0 +1 @@
"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"27","messages":"28","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":"29","usedDeprecatedRules":"18"},{"filePath":"30","messages":"31","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":"32","usedDeprecatedRules":"33"},"/Users/taylor/turing/4module/capstone/eras-fe/src/index.tsx",[],["34","35"],"/Users/taylor/turing/4module/capstone/eras-fe/src/reportWebVitals.ts",[],"/Users/taylor/turing/4module/capstone/eras-fe/src/Components/App.tsx",[],"/Users/taylor/turing/4module/capstone/eras-fe/src/Components/FormContainer.tsx",[],"/Users/taylor/turing/4module/capstone/eras-fe/src/Components/Question.tsx",[],"/Users/taylor/turing/4module/capstone/eras-fe/src/Components/LoginPage.tsx",["36"],"import React, { useEffect, useState } from 'react';\n const LoginPage = () =>{\n  let [userName,typeName] = useState('')\n\n  let [password,typePass] = useState('')\nlet createUserName = (event: React.ChangeEvent<HTMLInputElement>) =>{\n  typeName(event?.target.value)\n}\nlet createPass = (event: React.ChangeEvent<HTMLInputElement>) =>{\n  typePass(event?.target.value)\n}\n//are frontend validating this before we send it\n  return (\n    <section>\n    <form>\n\n      <input value = {userName} type = 'text' name = 'userName' placeholder=\"User Name\" onChange = {createUserName}\n      ></input>\n      <input value = {password} type = 'password' name = 'password' placeholder=\"Password\" onChange = {createPass}\n      ></input>\n    </form>\n    <button> Submit Me</button>\n    </section>\n  )\n}\nexport default LoginPage","/Users/taylor/turing/4module/capstone/eras-fe/src/Components/HeaderComponent.tsx",["37"],"import React, { useState } from 'react';\nimport { Link } from 'react-router-dom';\n\n\nconst HeaderComponent = () =>{\n  return (\n      <section>\n        {/* <img src = ''> logo*/}\n        <h1>Create My Calendar</h1>\n        <button >        \n          <Link to=\"/login\">Login</Link>\n        </button>\n      </section>\n  )\n}\nexport default HeaderComponent",["38","39"],{"ruleId":"40","replacedBy":"41"},{"ruleId":"42","replacedBy":"43"},{"ruleId":"44","severity":1,"message":"45","line":1,"column":17,"nodeType":"46","messageId":"47","endLine":1,"endColumn":26},{"ruleId":"44","severity":1,"message":"48","line":1,"column":17,"nodeType":"46","messageId":"47","endLine":1,"endColumn":25},{"ruleId":"40","replacedBy":"49"},{"ruleId":"42","replacedBy":"50"},"no-native-reassign",["51"],"no-negated-in-lhs",["52"],"@typescript-eslint/no-unused-vars","'useEffect' is defined but never used.","Identifier","unusedVar","'useState' is defined but never used.",["51"],["52"],"no-global-assign","no-unsafe-negation"]
```

Comment:
I'm not sure we need to push up .eslintcache. From what I've read, it looks like it can be ignored but I will do more research.